README.rst

```diff
@@ -39,3 +73,50 @@ documents in a hidden toctree near where they are referenced::
         :hidden:
 
         path/to/referenced-file.rst
+
+Getting Help
+************
+
+If you're having difficulties writing or testing, reach out to us on the
+`discussion forums`_. Be sure to categorize your question appropriately -
+if you're looking for technical help, the Development category is what you
+want. If you're stuck on an idea and could use some community help in
+bringing it to life, try the Collaborative Proposals category!
+
+.. _discussion forums: https://discuss.openedx.org/
+
+Code of Conduct
+***************
+
+All contributors to and reviewers of documentation in this repo must adhere
+to the `Open edX Code of Conduct`_. If you witness a breach of the Code of
+Conduct, please reach out to the maintainer of this repo or to the contacts
+listed in the Code of Conduct text.
+
+.. _Open edX Code of Conduct: https://openedx.org/code-of-conduct/
+
+People
+******
+
+See the `open-edx-proposals Backstage page`_ for more information about who the
+maintainers of this repo are and how to get in touch with them.
+
+.. _open-edx-proposals Backstage page: https://backstage.openedx.org/catalog?filters%5Bkind%5D=component&filters%5Buser%5D=all
```

Comment:
Link will be corrected once https://github.com/openedx/open-edx-proposals/pull/441 merges